ee/codegen/src/__test__/__snapshots__/generate-code.test.ts.snap

```diff
@@ -120,6 +120,39 @@ class InnerMapNode(MapNode):
 "
 `;
 
+exports[`generateCode > should generate code for %1 slack-message-trigger.ts > triggers/slack_message_trigger.py 1`] = `
```

Comment:
This was the main goal of the PR. In hindsight, I should have split it up as the trigger.py file in one PR, the workflow.py file in the other